packages/tables/src/Concerns/HasRecords.php

```diff
@@ -91,7 +91,7 @@ public function getTableRecordKey(Model $record): string
         $relationship = $this->getRelationship();
 
         $pivotClass = $relationship->getPivotClass();
-        $pivotKeyName = app($pivotClass)->getKeyName();
```

Comment:
This kills my (BelongsToMany)RelationManager when `$allowsDuplicates = true;`.

Reply:
Thanks for the heads up. I'll create a PR to remove the breaking changes.